### PR TITLE
Add custom SEO tags to web pages

### LINKED ISSUE #977: Alterar título das páginas
## Feature Description

As seguintes páginas estão todas com o mesmo título "Plataforma Sabiá".  

**Página Principal:**
Título: Plataforma Sabiá
Descrição: Encontre tecnologias e serviços voltados o semiárido brasileiro
Palavras-chave: semiárido, plataforma digital, plataforma sabiá, plataforma sabiá da ufersa, tecnologias para o semiárido, tecnologias sociais, inovação tecnológica, nordeste, universidades, pesquisas, startups, empresas, indústrias, inovação, invenção

**/announcements**
Título: Banco de Editais da Plataforma Sabiá
Descrição: Encontre uma lista atualizada de editais de incentivo à pesquisa e inovação
Palavras-chave: editais de pesquisa e inovação, inovação, semiárido, editais de fomento, financiamento de pesquisa, investimento em pesquisa, recursos para pesquisa

**/shopping-cart** _Essa página coloque como noindex_
Título: Carrinho de Compras da Plataforma Sabiá

**/search**
Título: Busca na Plataforma Sabiá
Descrição: Encontre tecnologias e serviços voltados o semiárido brasileiro
Palavras-chave: semiárido, plataforma digital, plataforma sabiá, plataforma sabiá da ufersa, tecnologias para o semiárido, tecnologias sociais, inovação tecnológica, nordeste, universidades, pesquisas, startups, empresas, indústrias, inovação, invenção

**/privacy-policy**
Título: Política de Privacidade da Plataforma Sabiá
Descrição: Conjunto de normas e informações para os usuários da Plataforma Sabiá
Palavras-chave: política de privacidade, normas da plataforma, usuários, regras, proteção de dados, lgpd

**/terms-of-use**
Título: Termos e Condições de uso da Plataforma Sabiá
Descrição: Conjunto de regras para uso da Plataforma Sabiá
Palavras-chave: termos de uso, terms of use, serviços, normas da plataforma, usuários, regras, proteção de dados, lgpd

**/solution/new**
Título: Cadastro de Soluções em Tecnologia ou Serviço
Descrição: Cadastre tecnologias ou serviços voltados para o semiárido
Palavras-chave: tecnologias, tecnologias sociais, inovação social, inovação, cadastro, solução, serviços, prestação de serviços no semiárido, semiárido, universidades, empresas, startups, invenção, formulário de cadastro

**/ideas**
Título: Banco de Ideias da Plataforma Sabiá
Descrição: Encontre uma lista de ideias para solução de problemas do semiárido
Palavras-chave: ideias, problemas, problemas em pesquisas, soluções, propor ideias, solução de ideias, ideias para invenções, cadastro de ideias, solução de problemas

**/about**
Título: Sobre a Plataforma Sabiá
Descrição: Conheça o que é a Plataforma Sabiá e como ela funciona
Palavras-chave: sobre, conhecer, conhecer a plataforma sabiá, sobre a plataforma sabiá, que é a plataforma sabiá da ufersa, quem criou a plataforma sabiá, o que tem a plataforma sabiá, funcionalidades, recursos, o que fazer na plataforma sabiá, tecnologias, serviços, semiárido

**Para cada página do detalhamento da tecnologia inserir:**

Link: _https://plataformasabia.com/t/%slug-da-tecnologia%_

Título: o titulo da tecnologia
Descrição: descrição da tecnologia
Palavras-chave: palavras-chave da tecnologia 

---------------

## Acceptance criteria

* 

## Implementation Brief

*

### PR DESCRIPTION
**Pull request by mateus4k**
## Summary

Resolves #977

- https://staging.plataformasabia.com/
- https://staging.plataformasabia.com/t/barragem-subterranea
- https://staging.plataformasabia.com/announcements

image

## Checklist

- [x] My code is tested and passes existing unit tests.
- [x] My code has an appropriate set of unit tests which all pass.
- [x] My code passes the linting.
- [x] My code has proper inline documentation.
- [x] I have manually tested this PR.

### CODE REVIEW

File: packages/web/pages/index.js [hunk 1]
```diff
@@ -27,6 +26,12 @@ const Home = ({ emailConfirmation, changeEmail, technologies, services }) => {
 
 	return (
 		<>
+			<Head
+				title={t('pages:home.title')}
+				description={t('pages:home.description')}
+				keywords={t('pages:home.keywords')}
+				ogImage={`${fullUrl}logo.svg`}
```
Comment (mateus4k) [inline packages/web/pages/index.js:33]: This defines the technology image in the url view

File: packages/web/pages/t/[technology]/index.js [hunk 2]
```diff
@@ -14,7 +14,12 @@ import {
 const Technology = ({ technology }) => {
 	return (
 		<>
-			<Head title={technology.title} />
+			<Head
+				title={technology.title}
+				description={technology.description}
+				keywords={technology.keywords.map((keyword) => String(keyword.term).trim())}
+				ogImage={technology.thumbnail?.url}
```
Comment (mateus4k) [inline packages/web/pages/t/[technology]/index.js:21]: Sets the URL of the technology image as a preview image

File: packages/web/pages/index.js [hunk 3]
```diff
@@ -25,8 +24,19 @@ const Home = ({ emailConfirmation, changeEmail, technologies, services }) => {
...
+		[fullUrl],
+	);
+
 	return (
 		<>
+			<Head
+				title={t('pages:home.title')}
+				description={t('pages:home.description')}
+				keywords={t('pages:home.keywords')}
+				ogImage={mainLogoUrl}
```
Comment (mateus4k) [inline packages/web/pages/index.js:38]: Sets the absolute URL of the Sabiá logo as the preview image

**Review by mtullyoc — APPROVED** [review]
Good job, @mateus4k!

We can make in a future some work with long tail keywords in case of weak SEO results.

**Pull request merged**